clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -4642,6 +4642,14 @@ class CIR_UnaryFPToFPBuiltinOp<string mnemonic, string llvmOpName>
   let llvmOp = llvmOpName;
 }
 
+def CIR_SqrtOp : CIR_UnaryFPToFPBuiltinOp<"sqrt", "SqrtOp"> {
+  let summary = "Floating-point square root operation";
+  
```

Comment:
Can you add a description with an example of the CIR format?

We're trying to standardize the descriptions of all our operations to include an example.